src/components/manage-productions/manage-productions.tsx

```diff
@@ -203,108 +225,123 @@ export const ManageProductions = () => {
       <StyledBackBtnIcon>
         <NavigateToRootButton />
       </StyledBackBtnIcon>
-      <DisplayContainerHeader>Manage Productions</DisplayContainerHeader>
-      <FormInputWrapper>
-        <FormInputWithLoader
-          onChange={(ev) => {
-            onChange(ev);
-            const pid = parseInt(ev.target.value, 10);
-            const confirmedPid = Number.isNaN(pid) ? null : pid;
-
-            setProductionIdToFetch(confirmedPid);
-            setShowDeleteDoneMessage(false);
-          }}
-          label="Production ID"
-          placeholder="Production ID"
-          name={name}
-          inputRef={ref}
-          onBlur={onBlur}
-          type="number"
-          loading={fetchLoader}
-        />
-        {delayOnGuideText && (
-          <StyledWarningMessage>
-            Please enter a production id
-          </StyledWarningMessage>
-        )}
-        {productionFetchError && (
-          <FetchErrorMessage>
-            The production ID could not be fetched. {productionFetchError.name}{" "}
-            {productionFetchError.message}.
-          </FetchErrorMessage>
-        )}
-      </FormInputWrapper>
-      <ErrorMessage
-        errors={errors}
-        name="productionId"
-        as={StyledWarningMessage}
-      />
-      {cachedProduction && (
-        <ShowListBtn
-          type="button"
-          onClick={() => {
-            setShowProductionsList(!showProductionsList);
-          }}
-        >
-          {showProductionsList ? "Hide" : "Show"} Productions List
-        </ShowListBtn>
-      )}
-      {(!cachedProduction || showProductionsList) && (
-        <PaginatedList
```

Comment:
This component is still in the project but not used anywhere. A problem now is also that there is no way of seeing all productions, the list in the scroll-list is only displaying the last 10 items. The same as the list on the landing-page. It's not pref to bulk-show all items, because we need some sort of limit to how many so some sort of "pagination"-view is needed. Could be that the list loads another 10 when reaching the bottom of the list.

![Screenshot 2024-12-10 at 15 12 21](https://github.com/user-attachments/assets/371f0fb5-cf0e-4142-978e-851a3e534de5)